packages/react-node/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@antv/g6-react-node",
   "description": "Using React Component to Define Your G6 Graph Node",
-  "version": "1.4.5-siren.4",
+  "version": "1.4.5-siren.5",
```

Comment:
Should we be incrementing if we didn't change the code in the package? I'd guess not since the versions aren't in sync anyway.